### DISCUSSION TITLE
Should intentionally stopped adapters through a job or via adapter overview still result in unhealthy health endpoint status?

### DISCUSSION THREAD

**Discussion by MLenterman** [frankframework/frankframework#6870] [General]
Should intentionally stopped adapters through a job or via adapter overview still result in unhealthy health endpoint status?

A simple use-case would be to stop processing of messages during a certain period of time through a job, by stopping the adapter the adapter during that period(Not the best use-case). Or possibly a production incident where you want to temporarily stop a specific adapter.

Currently stopping an adapter in any way will result in the pod getting restarted, because of unhealthy health probe in Kubernetes deployments

**Comment by nielsm5** [comment]
We're still working on this, in cloud environments this is indeed undesirable. 
Issue #4114 is still open in order to solve issues like this.

**Comment by philipsens** [comment]
This issue is also related: https://github.com/frankframework/frankframework/issues/3867

**Comment by nielsm5** [comment]
I hope this has been fixed in #7253.
server/health --> verifies all configuraties are started
configuration/name/health --> verifies all adapters are started
configuration/name/adapters/name/health --> detailed health info per adapter
